samples/files.py

```diff
@@ -75,6 +75,14 @@ def test_files_create_video(self):
         print(f"{result.text=}")
         # [END files_create_video]
 
+    def test_files_create_pdf(self):
+        # [START files_create_pdf]
+        model = genai.GenerativeModel("gemini-1.5-flash")
+        sample_pdf = genai.upload_file(media / "test.pdf")
+        response = model.generate_content(["Give me a summary of this pdf file.", sample_pdf])
```

Comment:
In other samples (e.g. video), the file is provided before the text in generate_content
Here the order is different; is there a significance to the order change?

Reply:
There's no particular significance to the order; you can list them in any order you want!

Reply:
AFAIK the only consideration for the order is that putting the file first will match the order you need to use if you want to cache the file and use it for a few questions.